test/clamav_freshclam.conf

```diff
@@ -0,0 +1,17 @@
+LogTime yes
+LogVerbose yes
+NotifyClamd /etc/clamav/clamd.conf
+Checks 24
+LogSyslog no
+DatabaseOwner clam
+# This option allows you to easily point freshclam to private mirrors.
+# If PrivateMirror is set, freshclam does not attempt to use DNS
+# to determine whether its databases are out-of-date, instead it will
+# use the If-Modified-Since request or directly check the headers of the
+# remote database files. For each database, freshclam first attempts
+# to download the CLD file. If that fails, it tries to download the
+# CVD file. This option overrides DatabaseMirror, DNSDatabaseInfo
+# and ScriptedUpdates. It can be used multiple times to provide
+# fall-back mirrors.
+# Default: disabled
+PrivateMirror http://clamav-mirror:8080
```

Comment:
This points the clamd container to the clamav-mirror container, without it clamav was pulling the signature database itself not through the clamav-mirror